.github/workflows/spoc-app.yml

```diff
@@ -33,5 +33,7 @@ jobs:
       run: black --check src --exclude spacy-lookup
     - name: Type checking with mypy
       run: mypy src --exclude spacy-lookup
+    - name: Download spaCy English Medium
```

Comment:
If this is a dependency as opposed to a lint or test, maybe move it up next to the Install Dependencies section?

Reply:
Done in latest commit